meta/src/manager.rs

```diff
@@ -0,0 +1,89 @@
+use crate::utils;
+use proc_macro2::TokenStream as TokenStream2;
+use quote::quote;
+use syn::visit::Visit;
+use syn::{Field, Ident, PathSegment};
+
+pub struct ManagerVisitor {
+    name: Ident,
+    functions: Vec<TokenStream2>,
+}
+
+impl ManagerVisitor {
+    pub fn new(name: &Ident) -> Self {
+        Self {
+            name: name.clone(),
+            functions: Vec::new(),
+        }
+    }
+
+    fn make_with(name: &Ident, segment: &PathSegment) -> TokenStream2 {
```

Comment:
If I didn't miss something, this fn is never used

Reply:
It was used to set `SyntaxMetadata::binop_border` before I removed it, but I think it's worth keeping anyway, in case we want to set native or simple cloneable fields in future.  And if we don't use it, it won't be linked into the executable.